spec/controllers/transcriptions_controller_spec.rb

```diff
@@ -170,7 +170,13 @@
       it 'contains an invalid attribute' do
         busted_params = { id: transcription.id, "data": { "type": "transcriptions", "attributes": { "garf": true } } }
         patch :update, params: busted_params
-        expect(response).to have_http_status(:unprocessable_entity)
+        expect(response).to have_http_status(:forbidden)
```

Comment:
```suggestion
        expect(response).to have_http_status(:bad_request)
```
... and ofc the test would be updated to match the 400 error thrown in the transcription `update` method.